tsdb/tsm1/file_store.go

```diff
@@ -377,6 +377,11 @@ func (t *fileTracker) ClearFileCounts() {
 }
 
 func formatLevel(level uint64) string {
+	// sanity check to avoid future bugs.
+	if level == 0 {
+		panic("TSM Files' Level '0' is not supported anymore.")
```

Comment:
We should never `panic` if we can avoid it, especially for a `level0` value here. If anything let's just log and move on but a panic I feel is too strong.